src/julia-parser.scm

```diff
@@ -620,15 +620,6 @@
 (define (space-before-next-token? s)
   (or (skip-ws (ts:port s) #f) (eqv? #\newline (peek-char (ts:port s)))))
 
-;; --- misc ---
-
-; Log a syntax deprecation, attributing it to current-filename and the line
-; number of the stream `s`
-(define (parser-depwarn s what instead)
```

Comment:
Are we sure we won't want this again at some point?

Reply:
No more deprecations!